### PR TITLE
feat(faucet): Add input field + network selection + FAQ section

### PR DESCRIPTION
**Pull request by jvmi7**
### Description

Adding in all the inputs for the faucet.
- Network radio card selection
- Address input
- FAQs from the old faucet (for now)

Screenshot 2024-06-03 at 4 13 08 PM

### CODE REVIEW

**Comment by jvmi7** [comment]
* **#366** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/ecosystem/366?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @jvmi7 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/dapp-console/app/faucet/components/FaucetContent.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  {
+    label: 'Ethereum Sepolia',
+    image: '/logos/eth-logo.png',
+    chainID: sepolia.id,
+  },
+  {
+    label: 'Fraxtal Sepolia',
+    image: '/logos/frax-logo.png',
+    chainID: 2523,
+  },
```
Comment (tremarkley) [inline apps/dapp-console/app/faucet/components/FaucetContent.tsx:30]: Frax Sepolia is no longer supported, so this should be removed

File: apps/dapp-console/app/faucet/page.tsx [hunk 2]
```diff
@@ -50,8 +54,20 @@ export default function Faucet() {
...
+        </CardContent>
       </Card>
+
+      <Tile
+        className="w-full max-w-screen-lg my-10"
+        variant="secondary"
+        title="Apply for bulk tokens →"
+        description="Superchain faucet only dispenses a small number of test ETH each day. Apply to get test tokens in bulk."
+        onClick={() => {}}
+      />
```
Comment (tremarkley) [inline apps/dapp-console/app/faucet/page.tsx:68]: This should be removed, we no longer provide a bulk testnet form

**Review by tremarkley — APPROVED** [review]

File: apps/dapp-console/app/faucet/page.tsx (hunk 2 above)
Comment (jvmi7) [reply apps/dapp-console/app/faucet/page.tsx:68]: dope, im removing this in the PR at the top of the stack to avoid merge conflicts

**Pull request merged**